draft-ietf-tls-dtls13.md

```diff
@@ -1174,69 +1174,88 @@ is received. (See {{dtls-epoch}} for the definitions of each epoch.)
 
 ##  DTLS Handshake Flights
 
-DTLS messages are grouped into a series of message flights, according
-to the diagrams below.
+DTLS handshake messages are grouped into a series of message flights. A flight starts with the 
+handshake message transmission of one peer and ends with the expected response from the 
+other peer. Table {{tab-flights}} contains a complete list of message combinations that consitute flights. 
+
+| Note | Client | Server | Handshake Messages |
+|-----+--------+--------+------------------ |
+|     |    x   |        | ClientHello |
+| (1) |        |    x   | HelloRetryRequest |
+|     |        |    x   | ServerHello, EncryptedExtensions, CertificateRequest*, Certificate*, CertificateVerify*, Finished | 
+| (2) |    x   |        | Certificate*, CertificateVerify*, Finished |
+|     |        |    x   | ACK | 
```

Comment:
(A client will send ACK in response to post-handshake messages, just not during the initial handshake.  I think this is okay as the table contents, but maybe we should ensure that the prose is clear this is just for the initial handshake.)  Though, given ekr's comment below, perhaps it's going away anyway and becomes irrelevant.

Reply:
I actually left ACK in because it was more confusing not to.